### DISCUSSION TITLE
Archive switch parameter in Remove-DuplicateItems.ps1

### DISCUSSION THREAD

**Discussion by Mark-Powerco** [David-Barrett-MS/PowerShell-EWS-Scripts#56] [General]
Firstly, thank you immensely for the Remove-DuplicateItems.ps1 script. It has saved me a lot of bother.

I have a question regarding the 'Archive' switch parameter. Correct me if I'm wrong, but there does not appear to be any references to the $Archive variable throughout the rest of the script. Reason I ask is that I am interested in scraping the 'In-place archive' mailbox for duplicate items, however I found the parameter had no effect.

Was this a capability that was intended to be built into the script but was never implemented/finished?

**Comment by David-Barrett-MS** [comment]
Apparently so!  I suspect the -Archive swtich only appeared as I've moved that to shared code.  I'll add archive support for this script.
